index.html

```diff
@@ -1136,10 +1136,13 @@ <h2>
             </ol>
           </li>
           <li>
-            <p>
+            <p data-cite="html">
               Present a user interface that will allow the user to interact
               with the |handlers|. The user agent SHOULD prioritize the user's
-              preference when presenting payment methods.
+              preference when presenting payment methods. It is RECOMMENDED
+              that the language of the user interface match |document|'s
+              [=document element|document element's=] [=Node/language=], if
```

Comment:
During the 15 April 2021 WPWG call, it was suggested to delete "if any".

Reply:
It's not true that a document will always have a language. it will have a language attribute, that language attribute might return the empty string.

Reply:
https://jsfiddle.net/54f8mLxz/7/